packages/strapi-plugin-content-type-builder/admin/src/components/AllowedTypesSelect/Text.js

```diff
@@ -0,0 +1,26 @@
+import styled from 'styled-components';
+import PropTypes from 'prop-types';
+
+const Text = styled.p`
```

Comment:
We will use the same `Text` as the one in the upload plugin but we should do this in another PR as we will use it in the entire content type builder.